epa-envirofacts-scraper/submodules/archiver.py

```diff
@@ -7,64 +7,61 @@
 ef_service_url = "http://iaspub.epa.gov/enviro/efservice"
 csv_url = "{}/{}/ROWS/{}:{}/CSV"
 default_dir = os.path.join(
-    os.path.dirname(__file__),
-    os.path.pardir,
-    os.path.pardir,
+    os.path.dirname(__file__), os.path.pardir, os.path.pardir,
     "epa_gov_enviro_facts_archive")
 
-
 def _mkdir(path):
     if not os.path.isdir(path):
         os.mkdir(path)
         logger.info("Created dir: %s", path)
 
-
 def _setup(f):
     def wrapper(*args, **kwargs):
-        _mkdir(os.path.realpath(kwargs.get('data_dir', default_dir)))
+        # set default dir if not passed in
+        kwargs['data_dir'] = kwargs.get('data_dir', None) or default_dir
```

Comment:
@danielballan Oh! neat. Will use that and probably open a new PR in the near future. Thanks.